pandas/core/arrays/timedeltas.py

```diff
@@ -397,6 +405,163 @@ def f(x):
 # ---------------------------------------------------------------------
 # Constructor Helpers
 
+def sequence_to_td64ns(data, copy=False, unit="ns", errors="raise"):
+    """
+    Parameters
+    ----------
+    array : list-like
+    copy : bool, default False
+    unit : str, default "ns"
+    errors : {"raise", "coerce", "ignore"}, default "raise"
+
+    Returns
+    -------
+    ndarray[timedelta64[ns]]
+    inferred_freq : Tick or None
+
+    Raises
+    ------
+    ValueError : data cannot be converted to timedelta64[ns]
+
+    Notes
+    -----
+    Unlike `pandas.to_timedelta`, if setting `errors=ignore` will not cause
```

Comment:
Still under discussion, but I think the standard we're moving to is to use single backticks for variables, arguments, attributes, functions, methods classes. And double backticks for values and code or pseudo-code. I think the `errors=ignore` should be double backticks.
```suggestion
    Unlike `pandas.to_timedelta`, if setting ```errors=ignore``` will not cause
```